maven-plugins/helidon-archetype-maven-plugin/src/it/projects/test1/archetype3/src/main/resources/META-INF/helidon-archetype.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<archetype-descriptor modelVersion="1.0" name="archetype2">
```

Comment:
Is this archetype v1 file here used to demonstrate that the validation is only made against XML files that have the proper schema declaration ? If yes, it's confusing ; can you instead make a dummy XML with a different name to avoid the confusion.

Reply:
This file is copied from other test and Validator does not validate this file because its root element is not equal to `archetype-script` as we use in version 2, so I do not understand why do I need to replace this file with other one.

Reply:
Mixing two different versions of the archetype XML is confusing.

This file is only used to show that it's not validated, its content is not used. Using a dummy file would re-enforce that it's only purpose is to not be validated, instead of having a fully valid V1 descriptor...